src/gt4py/next/common.py

```diff
@@ -141,6 +141,13 @@ def __and__(self, other: Set[Any]) -> UnitRange:
         else:
             raise NotImplementedError("Can only find the intersection between UnitRange instances.")
 
+    def __le__(self, other: Set[Any]):
```

Comment:
I added tests for __le__ and disabled the others, but will add them in a separate PR. It's better than undoing this change (because current implementation is broken) and less work...